static/app/components/replays/videoReplayerWithInteractions.tsx

```diff
@@ -0,0 +1,153 @@
+import type {Theme} from '@emotion/react';
+import {type eventWithTime, Replayer} from '@sentry-internal/rrweb';
+
+import {
+  VideoReplayer,
+  type VideoReplayerConfig,
+} from 'sentry/components/replays/videoReplayer';
+import type {ClipWindow, VideoEvent} from 'sentry/utils/replays/types';
+
+type RootElem = HTMLDivElement | null;
+
+interface VideoReplayerWithInteractionsOptions {
+  durationMs: number;
+  events: eventWithTime[];
+  onBuffer: (isBuffering: boolean) => void;
+  onFinished: () => void;
+  onLoaded: (event: any) => void;
+  root: RootElem;
+  start: number;
+  theme: Theme;
+  videoApiPrefix: string;
+  videoEvents: VideoEvent[];
+  clipWindow?: ClipWindow;
+}
+
+/**
+ * A wrapper replayer that wraps both VideoReplayer and the rrweb Replayer.
+ * We need both instances in order to render the video playback alongside gestures.
+ */
+export class VideoReplayerWithInteractions {
+  public config: VideoReplayerConfig = {
+    skipInactive: false,
+    speed: 1.0,
+  };
+  public iframe = {};
+  public videoReplayer: VideoReplayer;
+  public replayer: Replayer;
```

Comment:
Should these all be `public`? if we don't access them from outside of the class instance, then they should be `private`

Reply:
not accessed, i'll change to `private`